src/domain/community/user-lookup/user.lookup.service.ts

```diff
@@ -63,6 +63,33 @@ export class UserLookupService {
     return user;
   }
 
+  public async getUserByAuthId(
+    authId: string,
+    options?: FindOneOptions<User> | undefined
+  ): Promise<IUser | null> {
+    const user: IUser | null = await this.entityManager.findOne(User, {
+      where: {
+        authId,
+      },
+      ...options,
+    });
+
+    if (!user) {
+      this.logger.log(
```

Comment:
log to a particular level - warn/error/verbose. See more cases for this.

Reply:
Will do.